src/network/postgres_protocol_handler.cpp

```diff
@@ -28,6 +28,9 @@
 #include "type/value_factory.h"
 #include "network/marshal.h"
 #include "settings/settings_manager.h"
+#define SSL_MESSAGE_VERNO 80877103
+#define PROTO_MAJOR_VERSION(x) x >> 16
```

Comment:
Always put parens around your macro. (e.g. PROTO_MAJOR_VERSION(x) ((x) >> 16))

The reason for this is that a macro is literally a string substitution pre-compile time. If you omit parens operator precedence can get in your way and result in strange bugs. For example:

#define FOO(x) x * 16

and the expression FOO(3 + 4) will expand to 3 + 4 * 16 and result in 67 instead of 112.

or, let FOO(x) be x + 16, FOO(3) * 4 wll expand to 3 + 16 * 4 and equal to 67 instead of 76

Reply:
@tli2 Fixed. Thanks!